### PR TITLE
 Use String.startsWith in more places. 

### PR DESCRIPTION
**Pull request by XhmikosR**
Requires #4417 to be merged first and then I'll rebase.

### CODE REVIEW

File: lib/rules/function-url-no-scheme-relative/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const rule = function(actual) {
 			functionArgumentsSearch(decl.toString().toLowerCase(), 'url', (args, index) => {
 				const url = _.trim(args, ' \'"');
 
-				if (!isStandardSyntaxUrl(url) || url.indexOf('//') !== 0) {
+				if (!isStandardSyntaxUrl(url) || !url.startsWith('//')) {
```
Comment (XhmikosR) [inline lib/rules/function-url-no-scheme-relative/index.js]: This needs double checking but I assume this was the intention

Comment (ntwb) [reply lib/rules/function-url-no-scheme-relative/index.js]: If there's no test coverage it would be great to add some to be sure

Comment (XhmikosR) [reply lib/rules/function-url-no-scheme-relative/index.js]: It is covered already, but mistakes happen :)

**Comment by XhmikosR** [comment]
I will go through the codebase one more time and check for indexOf and such cases again later.

**Pull request merged**